src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -79,7 +80,7 @@ class ReferralController(
   }
 
   @GetMapping("/sent-referrals")
-  fun getSentReferrals(authentication: JwtAuthenticationToken): List<SentReferralDTO> {
+  fun getSentReferrals(authentication: JwtAuthenticationToken): List<SentReferralWithActionPlanIdDTO> {
```

Comment:
what's the reason this endpoint and `/sent-referral/{id}` return different DTOs?

Reply:
Will do - was worried we are implicitly changing the contract in other places by changing the SentReferralDTO response object - but the client should ignore an extra field

Reply:
yep 👍